xml_inscriptions/transcribed/KY.Lou.SAM.L.1929.17.34.xml

```diff
@@ -63,8 +63,8 @@
                   </summary>
                   <origin>
                      <!--Should contain information about origin of inscription including place and date.-->
-                     <date evidence="unspecified" notBefore="0001" notAfter="0200">c.
-                  first-second century CE</date>
+                     <date evidence="#palaeography #archaeology #formulae" notBefore="0001"
```

Comment:
the `@notAfter` seems a bit late if you're saying this is only a first c. CE inscription?